deploy/rbd/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -23,7 +23,7 @@ rules:
     verbs: ["get", "list", "watch", "update"]
   - apiGroups: [""]
     resources: ["configmaps"]
-    verbs: ["get", "list"]
+    verbs: ["get", "list","create","delete"]
```

Comment:
this can go away now since per @kfox1111 we can have node plugin as a sidecar and no longer need configmaps create/delete

Reply:
done